### ISSUE TITLE
"Install-Module PoShLog.Sinks.Syslog" throws "PackageManagement\Install-Package : No match was found"

### ISSUE THREAD

**Issue by gostega** [gitbute/PoShLog.Sinks.Syslog#1]
Thanks for the awesome tools.
----
`Install-Module PoShLog` < works ok
`Install-Module PoShLog.Sinks.Syslog` < throws:
```
PS C:\WINDOWS\system32> Install-Module PoShLog.Sinks.Syslog
PackageManagement\Install-Package : No match was found for the specified search criteria and module name
'PoShLog.Sinks.Syslog'. Try Get-PSRepository to see all available registered module repositories.
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:1809 char:21
+ ...          $null = PackageManagement\Install-Package @PSBoundParameters
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], Ex
   ception
    + FullyQualifiedErrorId : NoMatchFoundForCriteria,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage

```
- My Windows version: Windows 10 2004
- My powershell version:
```
PS C:\Users\me> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.19041.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by gitbute** [comment]
hey dude :)
i think you got to add '-AllowPrerelease' to the Install-Module command for my module, cause its still in preview.
tell me if it works.

**Comment by gitbute** [comment]
hey
i just published the 'real' 1.0.0 version of the module, so you can try installing again without the workaround.

**Comment by gitbute** [comment]
Also, going forward, please use this repository:
https://github.com/PoShLog/PoShLog.Sinks.Syslog

I will delete this one right now, as its not maintained here but in the link above.
